docs/index.md

```diff
@@ -20,30 +27,18 @@ The first iteration of OLM, termed OLM v0, included several concepts and feature
 Since its initial release, OLM has helped catalyse the growth of Operators throughout the Kubernetes ecosystem. [OperatorHub.io](https://operatorhub.io/)
 is a popular destination for discovering Operators, and boasts over 300 packages from many different vendors.
 
-# Why are we building OLM v1?
+## Why are we building OLM v1?
 
 OLM v0 has been in production for over 5 years, and the community to leverage this experience and question the initial
```

Comment:
#EditorialNotOrganizational

`community to leverage`?

I'm not sure what the original is attempting to express with this sentence.  Maybe 
>... for over 5 years, and this has provided the community with ample time to leverage this experience and question the initial goals ...

?

Reply:
Probably my fault. Sometimes I keep iterating over a paragraph to the point where it doesn't make sense anymore XD I tried fixing it up